app/api/users.py

```diff
@@ -0,0 +1,27 @@
+from fastapi import APIRouter, status
+from models.models import User, Token, UserUpdate, LoginRequest
+from service.users import create_user, update_user, remove_user
+from service.auth import login_for_access_token
+
+router = APIRouter()
+
+@router.post("/users/signup", status_code=status.HTTP_201_CREATED)
+def post_user(user:User):
+    result = create_user(user)
+    return result
+
+@router.post("/users/login", response_model=Token, status_code=status.HTTP_200_OK)
+async def login_user(login_request: LoginRequest):
+    result = login_for_access_token(login_request)
+    return result
+
+@router.patch("/users/{user_id}", status_code = status.HTTP_200_OK)
+def patch_user(user_id:str, user:UserUpdate):
+    result = update_user(user_id, user)
```

Comment:
```suggestion
def patch_user(user_id:str, user_update:UserUpdate):
    result = update_user(user_id=user_id, user_update=user_update)
```
p2: 꼭 파라미터를 명시할 필요가 없긴 한데 저희는 그렇게 사용하고 있어서 맞춰 주시면 좋을것 같아요~

Reply:
다른 코드도 위에 리뷰 처럼 변경해 주시면 좋을것 같습니다~~ user 보다는 좀 더 명확한 변수명이 좋을것 같아서 user_update 같이 해 주시면 좋을것 같습니다.

Reply:
넵! 수정하였습니다!